docs/vocascan-server/traefik/traefik.md

```diff
@@ -0,0 +1,165 @@
+# Installation with Traefik
+
+This guide will get you through the installation of your own vocascan-server paired with Traefik as a reverse proxy and
+Let's Encrypt.
+
+1. First of all you have to install Docker an Docker-Compose. To do this have a look at point 1 and 2 our
+   [guide](vocascan-server/installation#_2-Docker) in the general vocascan-server installation section.
+
+2. After that install `apache2-utils` to create a secure password hash for the Traefik monitoring website with
+   `htpasswd`
+
+   ```bash
+   sudo apt-get install apache2-utils
+   ```
+
+   Create password hash
+
+   ```bash
+   htpasswd -nb admin my_secure_password
+   ```
+
+   The output will look something like this. Store it for the configuration later
+
+   ```
+   admin:$apr1$ppHrFJc4$3/iv8jBNSDXlxiAuP1GDH1
+   ```
+
+3. Download the vocascan-server repository
+
+   ```bash
+   git clone https://github.com/vocascan/vocascan-server.git
+   ```
+
+4. Copy Traefik and Docker configs from cloned repo
+
+   copy Traefik folder in you current directory
+
+   ```bash
+   cp -R vocascan-server/docker/traefik traefik
+   ```
+
+   Change directory
+
+   ```bash
+   cd traefik
+   ```
+
+5. Create `acme.json` to store certificate information
+
+   ```bash
+   touch acme.json
+   ```
+
+   - give root access to file
+
+   ```bash
+   chmod 600 acme.json
+   ```
+
+6. Open `vocascan.config.js` to configure you server
+
+   ```bash
+   nano vocascan.config.js
+   ```
+
+   For more details about the configuration see [configuration](vocascan-server/configuration) page.
+
+7. Open `traefik.toml`
+
+   ```bash
+   nano traefik.toml
+   ```
+
+   and update the email prop to you email
+
+   ```toml
+                        ...
+   14    [certificatesResolvers.lets-encrypt.acme]
+   15       email = "your@mail.com"
+   16       storage = "acme.json"
+                        ...
+   ```
+
+8. Open `traefik_dynamic.toml`
+
+   ```bash
+   nano traefik.toml
+   ```
+
+   - Add the secret key (line 3), you created in the beginning
```

Comment:
* `in step 2` (with hyperlink)

Reply:
I can't create a hyperlink to the 2. step

Reply:
oh, you're right. that s not that easy. If you like that idea with the hyperlink, thats even possible by adding an anchor with an a tag. But If you dont like it, just resolve this conversation.